PathFinder.cc

```diff
@@ -395,21 +511,77 @@ void untilFound() {
   }
 }
 
-void disposeVoxelCache() {
-  voxels.clear();
-  voxelo.clear();
+void detectDestGlobal(Vec *position, int detectDir) {
+  if (iterDetect >= maxIterDetect) {
+    return;
+  }
+  iterDetect++;
+
+  bool collide = detect(position, true);
+
+  if (detectDir == 0) {

```

Comment:
Sounds like `detectDir` should be an `enum` of choices to make the code more clear?

Reply:
https://github.com/webaverse/app-wasm/pull/35/commits/ca7fa31e931dd475e9db7451da988915fddb4473